data/changelog.txt

```diff
@@ -15,235 +15,41 @@ Add support for json-defined achievements to provide more goals to strive for.
-Update for .410 and .454 firearms longest_length
-Update for 9x18 and 762x25 firearms longest_length
-New scenario: Fungal Infected
-Moves and updates 2 Fuji Structures stores to mainline.
-Moves and updates 2 Fuji Structures apartments to mainline.
-Add stereos to most cars, trucks and vans
-Second batch of ten furniture JSONs that I've edited. More to come.
-Remove laserturret
-Moves and updates Fuji Structures light industry special to mainline.
-Make nuclear reactor and nukes give plutonium
-Removes RM-99 caseless revolver
-New starting trait: Fast Reflexes
-More Pumpkin recipes
-Fixes egg spawn lists
-Moves and updates Fuji Structures airport to mainline.
-Adds new martial art: Desert Wind
-Typo in description puts it at Seven-Round clip whilst it's actually an Eight round
-Add sand castle construction
-Allow the of use grappling hooks on ledges
+Mi-go, masters, and necromancers can open doors.
+Adds a bulk shipping crate possibility to the supplydrop map extra.
+New zombie: necro-boomer.
+Overhaul the mall.
 
 ## Interface:
 Introduce and migrate most windows to ui_adaptor.
 Crafting GUI - Highlight hidden recipe info
-Minor uilist code clean-up
-Migrate overmap_ui.cpp to ui_adaptor
-Migrate inventory_ui.cpp and game_inventory.cpp to ui_adaptor
-Ask for confirmation on eating raw meat
-Tweak interface colors when removing vehicle parts
-Nested items that are not count_by_charges stack in menus
-When aiming, draw approximate line of fire for each turret in range
-Block bound inventory keys from auto assignment
-Compact aim window pt.2 - Numbers
-Add resizing callbacks to uilist
-Migrate message log UI and NPC trade UI to ui_adaptor
-adjust insert menu to support multiple items
-Reduce calorie message spam to once every 12 hours
-Migrate debug map editor to ui_adaptor
-Added more options for view movement in target ui
-Migrate inventory item menu, scores UI, and armor sorting menu to ui_adaptor
-Displays carrion spawn rate in world options as factor
-Migrate advanced inventory menu and defense game menus to ui_adaptor
-Hide main zone manager UI when selecting zone area
-Migrate iuse games to ui_adaptor
-Make unloading items not pick them up
-Place wished for liquids in containers by default
-Sepia Dark overlay for display of map memory
-Migrate mouse view and NPC dialogue window to ui_adaptor
-Custom map memory color overlay option
-Rename Deathblow to Mordhau
-Add terrain overlay callbacks and migrate targeting UI to ui_adaptor
-Clarify movement cost from leg/feet encumbrance doesn't apply only to running
-Migrate the '@' menu to ui_adaptor
-Overhaul look around panels
-Make debug and world option tabs switch places
-Fix the display error of ammo capacity
-Adjust the display content of kmenu
-Allow traversing main menu horizontally with TAB
-Migrate vehicle interaction menu to ui_adaptor
+When aiming, draw approximate line of fire for each turret in range.
```

Comment:
```suggestion
Overhaul target selection and aiming UI.
```
That pretty much summarizes a bunch of UI changes by me (#39785, #40198, some bugfixes) and @Kilvoctu (#37715, #40236)